Add "Get Started with Bytebase" (#213)
bytebase/bytebase.com: content/docs/en/get-started-with-bytebase/prepare-the-workspace.md

---
title: Prepare the Workspace
---
## Manage members

> Click `Settings` on the top bar to manage members in your workspace. Here your role is set as Owner by default. You can add new members by email.  

Reviewer: ```suggestion
Click `Settings` on the top bar to manage members in your workspace. You are assigned the `Owner` role as the first user and can add new members by email.  
```

![_](/static/docs-assets/add-users.png)  

![_](/static/docs-assets/manage-user.png)  

> >This demo uses the free version of Bytebase. It doesn't support user role management, and the user becomes workspace owner by default.  

Reviewer: ```suggestion
>This demo uses the free version of Bytebase. It doesn't support role management (RBAC), and all users assume the `Owner` role.  
```

> >With team or enterprise plan, you can manage users' roles in the workspace and in a project.  

Reviewer: ```suggestion
>With team or enterprise plan, you can manage users' roles in the workspace and project level.  
```
